test/e2e/elasticsearch_test.go

```diff
@@ -188,9 +188,6 @@ func getJaegerSimpleProdWithServerUrls(name string) *v1.Jaeger {
 	if specifyOtelImages {
 		logrus.Infof("Using OTEL collector for %s", name)
 		exampleJaeger.Spec.Collector.Image = otelCollectorImage
-	}
-
-	if specifyOtelConfig {
```

Comment:
Can you also apply this change in elf_provisioned_elasticsearch_test self_provisioned_elasticsearch_kafka_test sidecar_test and streaming_test?